python/tests/spark/test_functions.py

```diff
@@ -30,8 +30,8 @@ def test_areas(self):
         """Test calculating bounding box's area."""
         df = self.spark.createDataFrame(
             [
-                (Box2d(1, 2, 1.0, 1.0),),
-                (Box2d(10, 12, 1.0, 5.0),),
+                (Box2d(1, 2, 2.0, 3.0),),
```

Comment:
maybe add a couple of basic cases to test the udf?

Reply:
done